plugins/sep/varnish/varnish.cc

```diff
@@ -26,18 +26,33 @@ void Varnish::setView(ViewInterface::WeakPtr viewWeak) {
   this->viewWeak = viewWeak;
 }
 
+void Varnish::resetView(ViewInterface::WeakPtr viewWeak) {
+  this->viewWeak = viewWeak;
+
+  gdk_threads_enter();
+  GtkWidget *newBox = gtk_vbox_new(false, 0);
+  for (GList *iter = gtk_container_get_children(GTK_CONTAINER(box));
+       iter != nullptr; iter = iter->next) {
+    gtk_widget_reparent(GTK_WIDGET(iter->data), newBox);
+  }
+  box = newBox;
+  viewWeak.lock()->addSideWidget("Varnish", box);
+  gdk_threads_leave();
+}
+
 static void varnish_toggled(GtkToggleButton *button, gpointer varnishP) {
+  (void)button;
   // Have a member function sort out the varnishState
   static_cast<Varnish *>(varnishP)->fixVarnishState();
   // Force a redraw when varnish is toggled.
-  static_cast<Varnish *>(varnishP)->forceRedraw();
+  static_cast<Varnish *>(varnishP)->triggerRedraw();
 }
-
+/*
 void Varnish::forceRedraw() {
   ViewInterface::Ptr view(viewWeak);
   view->invalidate();
 }
-
+*/
```

Comment:
Better just remove it, instead of commenting it out. We use versioning for a reason ;-)

Reply:
Oops, forgot to remove it. Fixed